google/cloud/spanner_v1/services/spanner/client.py

```diff
@@ -28,6 +28,7 @@
     Type,
```

Comment:
Please see the [history](https://github.com/googleapis/python-spanner/commits/main/google/cloud/spanner_v1/services/spanner) of directory `google/cloud/spanner_v1/services/spanner` which shows that this file is autogenerated. Please propose changes to [GAPIC generator python](https://github.com/googleapis/gapic-generator-python) instead. 

Any handwritten changes to files in directory `google/cloud/spanner_v1/services/spanner` will be reverted in the next automated owlbot PR.